src/commponents/Footer.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+enum Status {
+  Active = 'active',
+  Completed = 'completed',
+  All = 'all',
+}
```

Comment:
Recommendation: Move the enum Status out of the App and import it where needed instead of duplicating the code.